lib/Fmt_ast.ml

```diff
@@ -76,28 +76,25 @@ let protect =
 let update_config ?quiet c l =
   {c with conf= List.fold ~init:c.conf l ~f:(Conf.update ?quiet)}
 
+let fmt_elements_collection ?(first_grp = true) ?(last_grp = true)
```

Comment:
Shouldn't we call these `first_sep` and `last_sep` ?

Reply:
We can, I've been struggling to find a good name for these parameters.